store/iavl/store.go

```diff
@@ -71,11 +78,11 @@ func UnsafeNewStore(tree *iavl.MutableTree) *Store {
 // GetImmutable returns a reference to a new store backed by an immutable IAVL
 // tree at a specific version (height) without any pruning options. This should
 // be used for querying and iteration only. If the version does not exist or has
-// been pruned, an error will be returned. Any mutable operations executed will
-// result in a panic.
+// been pruned, an empty immutable IAVL tree will be used.
+// Any mutable operations executed will result in a panic.
```

Comment:
I wonder why the behavior here needs to change? Ah, for the historical queries once the new, empty store is created.